packages/drivers/odsp-driver/src/zipItDataRepresentationUtils.ts

```diff
@@ -0,0 +1,496 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert, IsoBuffer, Uint8ArrayToString } from "@fluidframework/common-utils";
+
+/** Default encoding for strings */
+const stringEncoding = "utf-8";
```

Comment:
given format is only using utf-8, I'd inline this variable - results in slightly smaller bundle size (only 2 usages).